Veeam/PRTG-VeeamBRStats.ps1

```diff
@@ -36,6 +35,26 @@ param(
   
 )
 
+
+
+#############################################################################
+#If Powershell is running the 32-bit version on a 64-bit machine, we 
+#need to force powershell to run in 64-bit mode .
+#############################################################################
+if ($env:PROCESSOR_ARCHITEW6432 -eq "AMD64") {
+    write-warning "Y'arg Matey, we're off to 64-bit land....."
+    if ($myInvocation.Line) {
+        &"$env:WINDIR\sysnative\windowspowershell\v1.0\powershell.exe" -NonInteractive -NoProfile $myInvocation.Line
+    }else{
+        &"$env:WINDIR\sysnative\windowspowershell\v1.0\powershell.exe" -NonInteractive -NoProfile -file "$($myInvocation.InvocationName)" $args
+    }
+exit $lastexitcode
+}
+
+
+
+
+
```

Comment:
Section does not work as expected. 

If executed on 32Bit PowerShell Promt, Code is uncompleted.
`PS D:\> & '.\PRTG Veeam Advanced Sensor.ps1
>>
>>
>>
>>
>>`